diary-alexander-jaffray/modernized/15-notes-diary.adoc

```diff
@@ -1352,12 +1395,17 @@ the year of our Lord one thousand six hundred and fifty seven years.
 By warrant signed be the Commissioners of the exchequer.`"
 +++[+++Endorsement]
 
+[.signed-section-context-close]
```

Comment:
Lets mark this a `[.signed-section-closing]` instead of a `[.signed-section-context-close]`. The [.signed-section-context-close] are generally for dates or locations that are put BELOW the signature. But when it is put above the signature, it leaves a big space. Not sure if you can see that in the PDF.